### DISCUSSION TITLE
Stonecutter only hurt players, not mobs, so they cant be used for mob farms

### DISCUSSION THREAD

**Discussion by TomLewis** [PurpurMC/Purpur#1310] [__Completed__] [answered]
I love the idea of the stonecutter working, but players abuse it with mob farms, can we can an option to disable them hurting mobs? just hurt players
```
      stonecutter:
        damage: 1.0
```

**Comment by RhythmicSys** [comment, marked as answer]
This is actually a feature of [PurpurExtras](https://github.com/PurpurMC/PurpurExtras/) - it's currently set up to configure a blacklist, but if it's requested on that page, it wouldn't be too difficult to switch between a blacklist and whitelist
